js/models/order/Order.js

```diff
@@ -47,6 +47,15 @@ export default class extends BaseModel {
       // convert price fields
       response.contract.buyerOrder.payment.amount =
         integerToDecimal(response.contract.buyerOrder.payment.amount, true);
+
+      // if (response.contract.disputeResolution) {
```

Comment:
This will be needed once the server is properly providing the payout amount. I'll uncomment it then.